app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -89,36 +86,23 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     window.addEventListener('message', this.receiveMessage.bind(this), false)
   }
 
-  CookieBanner.prototype.receiveMessage = function (event) {
-    var trustedDomain = 'publishing.service.gov.uk'
-    var origin = event.origin
+  CookieBanner.prototype.isInCookiesPage = function () {
+    return window.location.pathname === '/help/cookies'
+  }
 
-    // Return if no origin is given or the browser doesn't support lastIndexOf
-    if (!origin || !origin.lastIndexOf) {
-      return
-    }
+  CookieBanner.prototype.isInIframe = function () {
+    return window.parent && window.location !== window.parent.location
+  }
 
-    // Polyfill origin.endsWith(trustedDomain) for IE
-    var offset = origin.length - trustedDomain.length
-    var trustedOrigin = offset >= 0 && origin.lastIndexOf(trustedDomain, offset) === offset
+  CookieBanner.prototype.parentIsPublishingDomain = function () {
+    var publishingDomain = 'publishing.service.gov.uk'
```

Comment:
It would be good if this worked the same way in the dev environment too :-).

Reply:
Removed the `parentIsPublishingDomain` requirement based on Kevin's suggestion.